CONTRIBUTING.md

```diff
@@ -191,13 +259,13 @@ Local config files will be ignored by default in git.
 For example, to create an 'alice' profile locally, put a file called `local-alice.json` in the
 `config` directory:
 
-```
+```json
 {
-  "storageProfile": "alice-profile",
+  "storageProfile": "alice-profile"
 }
 ```
 
-This will then set the `userData` directory to `%appData%/Session-alice-profile` when running the `alice` instance.
+This will then set the `userData` directory to `[PROFILE_PATH]//Session-alice-profile` when running the `alice` instance.
```

Comment:
do we support this at all?

Reply:
Not sure. Just updated the docs. Will verify tomorrow.

Reply:
Confirmed no longer needed. Fixed